proposals/PluginsAudit2015/audit.md

```diff
@@ -0,0 +1,223 @@
+# Cordova Plugins
+
+**NOTE**: Data for this proposal was collected on October 28th, 2015. 
+
+## Battery-Status
+* 1 Month Downloads: 5116
+* Open Issues: 18
+* Open Pull Requests: 3
+* Spec: http://www.w3.org/TR/battery-status/ 
+* android 5 has the ability to grab battery info in uiwebview built in
+* isPlugged is useful but not being used much
+* Recommendation: Deprecate! 
+
+## Camera
+* 1 Month Downloads: 17503
+* Open Issues: 85
+* Open Pull Requests: 20
+* Recommendation: Keep. Continue merging PRs and avoiding adding too many new features
+
+## Console
+* 1 Month Downloads: 45894
+* Open Issues: 5
+* Open Pull Requests: 1
+* console output should show up in Xcode by default. 
+* Recommendation: Move functionality back into platforms
+
+## Contacts
+* 1 Month Downloads: 6096
+* Open Issues: 71
+* Open Pull Requests: 10
+* Spec: http://www.w3.org/TR/contacts-api/ (discontinued)
+* API needs to be redone. Currently brittle and many platform specific quirks
+* Recommendation: Keep. Leave it as is and revist if new Contacts API is published by W3C. Api does need to be redone, but low priority.
+
+## Device
+* 1 Month Downloads: 53695
+* Open Issues: 7
+* Open Pull Requests: 1
+* Refactor: Update interface to be async?
+* Maybe keep it the same
+* Recommendation: Keep. Discuss if it makes sense to add it to hello world template as a default plugin. Add it to phonegap template. Add it as a dependency on tests that need it (Device-motion). Discuss if switching api async makes sense.
+
+## Device Orientation (compass)
+* 1 Month Downloads: 3823
+* Open Issues: 8
+* Open Pull Requests: 3
+* No Spec
+* maybe combine device-motion + compass?
+* Needs better API
+* rename to cordova plugin compass? 
+* Recommendation: Keep. Consider renaming plugin to cordova-plugin-compass. Needs improved API but low priority.
+
+## Device Motion (Accelerometer)
+* 1 Month Downloads: 3886
+* Open Issues: 8
+* Open Pull Requests: 0
+* spec http://www.w3.org/TR/orientation-event/
+* Starting to become available in browsers http://caniuse.com/#search=orientation 
+* rename to cordova-plugin-device-orientation-motion?
+* low priority: polish code, update code to remove accelerometer references. 
+* Recommendation: Keep. Code needs a cleanup but low priority.
+
+## Dialogs
+* 1 Month Downloads: 12456
+* Open Issues: 13
+* Open Pull Requests: 17
+* Using custom styled divs should be used instead of alerts
+* In HTML 5.1 there is a proposed dialog element which actually seems pretty good. http://www.w3.org/html/wg/drafts/html/master/single-page.html#the-dialog-element 
+* looks good on android
+* looks ugly on ios
+* currently not sync which is how default alerts work 
+* window.alert still available but says index.html
+* Recommendation: Deprecate! Encourage use of HTML/CSS/JS divs instead.
+
```

Comment:
I'd argue against this. While it is common for UI frameworks to include a dialog element, being able to use the default device dialog is handy and should be kept. One line to show a dialog is *incredibly* useful compared to N lines of CSS and JS.